politeiawww/cmd/politeiawwwcli/commands/users.go

```diff
@@ -0,0 +1,14 @@
+package commands
+
+type UsersCmd struct {
```

Comment:
Right now you have these as optional flags, I would probably set them as optional arguments instead.  

```suggestion
type UsersCmd struct {
	Args struct {
		Email    string `long:"email" description:"Email query"`
		Username string `long:"username" description:"Username query"`
	} `positional-args:"true"`
}
```

Reply:
Yeah, I see your point.  Using makes more sense in this context.